apps/server/src/agents/ZeroChatAgent.ts

```
@@ -0,0 +1,103 @@
+ * – Exposes a WebSocket interface handled by the base class.
+ * – Provides an HTTP API:
+ *   • GET  /history            → full message history as JSON
+ *   • POST /chat { message }   → send a user message and stream the assistant reply
+ */
+export class ZeroChatAgent extends AIChatAgent<typeof env> {
+  /**
+   * Generate the assistant reply whenever we receive a new user message.
+   * The base class already streams data to connected clients; we only need to
+   * return a Response to the originator.
+   */
+  async onChatMessage(onFinish: StreamTextOnFinishCallback<{}>): Promise<Response> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)